frontend/packages/console-shared/src/components/shortcuts/Shortcut.tsx

```diff
@@ -51,7 +54,9 @@ const Shortcut: React.FC<ShortcutProps> = ({
         )}
         {keyName && (
           <Command>
-            {keyName.length === 1 ? keyName.toUpperCase() : _.startCase(keyName.toLowerCase())}
+            {keyName.length === 1 || (keyName.length === 2 && keyName !== 'Up')
```

Comment:
This change seems unnecessary.

Reply:
The current code `{keyName.length === 1 ? keyName.toUpperCase() : _.startCase(keyName.toLowerCase())}` adds a space between `F 1`  This was a workaround, is their a better way?
<img width="998" alt="Screen Shot 2021-02-02 at 10 01 34 AM" src="https://user-images.githubusercontent.com/1874151/107428988-63441180-6af1-11eb-810a-2b50e5a0ca4f.png">

Reply:
oh i didn't realize `startCase` did that. 
We we want then is to use `upperFirst` instead:
`{keyName.length === 1 ? keyName.toUpperCase() : _.upperFirst(keyName.toLowerCase())}`

Reply:
updated